raiden/raiden_event_handler.py

```diff
@@ -315,6 +338,12 @@ def handle_contract_send_channelsettle(
         # be a race condition when both nodes try to settle
         # at the same time.
         pass
+    except ChannelOutdatedError as e:
+        log.debug(
+            f'{e.args}. '
+            f'Current channel identifier: {e.current_channel_id}, '
```

Comment:
I'd prefer the two channel ids to be keyword arguments. This would be more like we use structlog in general.